src/test/resources/converter.yml

```diff
@@ -1 +1,5 @@
 nominatimURL: https://nominatim.openstreetmap.org/search/
+
+openCageDataURL: https://api.opencagedata.com/geocode/v1/json
+openCageDataKey: XXXXXXXX
```

Comment:
I could in theory commit my personal ocd key here, it's a free key and is pretty limited anyway, but that way we could actually test ocd in travis. Like this the test will always fail.

Reply:
WDYT @karussell?

Reply:
Ok, let's try. Or we can also try to hide it and then it will work on travis only?

Reply:
This should not be a problem for now.